.github/workflows/ci.yml

```diff
@@ -31,12 +31,12 @@ jobs:
         run: npm run test:ci
```

Comment:
could you please tell me - why the `workflow` changed to `workflowS` and what is the reason of these changes?

Reply:
It wasn't working with `workflow` dir. I mean that CI was always broken here :( So I fixed it